server.js

```diff
@@ -17,7 +17,7 @@ const hbs = exphbs.create({ helpers });
 const sess = {
   secret: 'Super secret secret',
   cookie: {
-    maxAge: 3600,
+    maxAge: 60 * 60 * 1000, // One hour!
     httpOnly: true,
     secure: false,
     sameSite: 'strict',
```

Comment:
I only fixed a bit of whitespace in auth.js, and for some reason GitHub has troubles showing it below.